docs/websites.md

```diff
@@ -229,14 +232,18 @@ resources:
 
 Feel free to customize the `/asset/` path. If your application is a JS application backed by a PHP API, you will want to invert API Gateway and S3 (set S3 as the `DefaultCacheBehavior` and serve API Gateway under a `/api/` path).
 
-> The first deployment takes a lot of time (20 minutes) because CloudFront is a distributed service. The next deployments that do not modify CloudFront's configuration will not suffer from this delay.
+> The first deployment takes a lot of time (20 minutes) because CloudFront is a distributed service. The next deployments that do not modify CloudFront's configuration will not suffer from this delay. You will know it is ready because the `Status` column will change from `In Progress` to `Deployed` on your [CloudFront dashboard](https://console.aws.amazon.com/cloudfront/home?region=eu-west-1#)
```

Comment:
```suggestion
> The first deployment takes a lot of time (20 minutes) because CloudFront is a distributed service. The next deployments that do not modify CloudFront's configuration will not suffer from this delay. You will know it is finished when the `Status` column changes from `In Progress` to `Deployed` in your [CloudFront dashboard](https://console.aws.amazon.com/cloudfront/home).
```

Reply:
@mnapoli The URL with the `?region=eu-west-1` I got it from here, a bit further down. Should I hange that too?

Reply:
Yes that would be perfect 👍

Reply:
Done :-)